### PR TITLE
JIRA: ABC-220 Alembic prefabs created before 2.2.0-pre.3 get "switched" to External references

### PR DESCRIPTION
**Pull request by clusty**
Fix is done during deserialization.

### CODE REVIEW

**Review by judubu — APPROVED** [review]
👍

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed bug that caused previous Alembic instance in prefabs to get converted to out of project references.
```
Comment (sebastienduverne) [inline com.unity.formats.alembic/CHANGELOG.md:11]: Suggestion:
`- Fixed a bug that was converting previous Alembic instance in prefabs into out-of-project references.`

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2021.2.0a15 (Windows only)

**Review by sebastienduverne — APPROVED** [review]
Changelog OK.

**Pull request merged**